File: AnonymizerCore/ApplicationCore/Config/TableConfig.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Scrambler.Config
{
    public class TableConfig
    {
        public string FullTableName { get; set; }
        public List<ScrambledColumnConfig> ScrambledColumns { get; set; }
        public List<ConstantColumnConfig> ConstantColumns { get; set; }
        public List<List<string>> PairedColumnsInsideTable { get; set; }
        public List<PairedColumnsOutsideTableConfig> PairedColumnsOutsideTable { get; set; }
    }
}

File: anonymizer_ui/src/app/demo/domain/databaseConfig/tableConfig.ts
import { ScrambledColumnConfig } from './scrambledColumnConfig';
import { ConstantColumnConfig } from './constantColumnConfig';
import { PairedColumnsOutsideTableConfig } from './pairedColumnsOutsideTableConfig';

export interface TableConfig {
    fullTableName : string;
    where : string;
    scrambledColumns : Array<ScrambledColumnConfig>;
    constantColumns : Array<ConstantColumnConfig>;
    pairedColumnsInsideTable : Array<Array<string>>;
    pairedColumnsOutsideTable : Array<PairedColumnsOutsideTableConfig>;
}